src/gui/application.cpp

```diff
@@ -448,6 +448,9 @@ void Application::parseOptions(const QStringList &options)
             _debugMode = true;
         } else if (option == QLatin1String("--version")) {
             _versionOnly = true;
+        } else if (option.endsWith(".owncloud")) {
```

Comment:
I'd suggest to put this behind a `--downloadplaceholder` switch

Reply:
Then it is harder to just associate the .owncloud extension with the program.